vmware/ngc/NGC-Plugin/src/test/java/service/impl/DatastoreTest.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * 
+ */
+
+package service.impl;
```

Comment:
Can we maintain the same package structure as source , like  org.opensds.vmware.ngc.service.impl;